contrib/patterns/_grid-media.scss

```diff
@@ -0,0 +1,16 @@
+$medium-screen: 1000px;
+$medium: "only screen and (min-width:#{$medium-screen})";
```

Comment:
It feels weird putting all of this in a variable? It could also get repetitive.

I wonder how we could make this a more succinct API, and maybe have some sane defaults?

Could some of this go within the default grid settings map?